cost/billing_center_cost_anomaly/billing_center_cost_anomaly.pt

```diff
@@ -6,13 +6,11 @@ long_description ""
 severity "high"
 category "Cost"
 tenancy "single"
-default_frequency "daily"
```

Comment:
Is there a reason you removed the default_frequency?

Reply:
I did not mean to remove that.